src/compass/utils/h5_helpers.py

```diff
@@ -617,7 +617,13 @@ def corrections_to_h5group(parent_group, burst, cfg, rg_lut, az_lut,
                  f'Ionospheric delay (range) {desc}',
                  {'units': 'meters'}),
         ]
-        if weather_model_path is not None:
+
+        if weather_model_path is None:
+            correction_items.append(Meta('static_los_troposphere_delay',
+                                             ds.GetRasterBand(5).ReadAsArray(),
+                                             f'Static LOS troposphere delay {desc}',
```

Comment:
I would not include the static troposphere as a correction layer inside the product. This correction can be easily calculated using the static layers product. So, there should be no need to have it there too

Reply:
Got it. I've changed the static troposphere LUT not to be written into the product.